### PR TITLE
Add PWD examples for multiple platforms.

### LINKED ISSUE #2275: Documentation docker run examples for current directory for all platforms
# Docker run examples for current directory for all platforms

It would be nice to get examples for how to mount the current directory on multiple platforms, Linux, MacOS, PowerShell, Windows.

Ref `docker run ...` example on https://www.zaproxy.org/docs/docker/full-scan/ with code in https://github.com/zaproxy/zaproxy-website/blob/main/site/content/docs/docker/full-scan.md?plain=1#L55

### PR DESCRIPTION
**Pull request by sastorsl**
Add `docker run ...` examples when mounting the current directory as a volume in the container.

The examples show how to do this on Linux/MacOS (as before), PowerShell and Windows.

Resolves #2275

### CODE REVIEW

**Comment by sastorsl** [comment]
> Would it be better to include the details in a new section on https://www.zaproxy.org/docs/docker/about/ and then just link to that from the specific packaged scan pages?

I updated the PR to that effect.
How does it look?

**Comment by psiinon** [comment]
How about having a new section on the 'ZAP Docker User Guide', mybe immediatelly under 'Usage Instructions':
```
### Mounting Directories

I many cases you will need to mount a local directory in Docker...
```
And then include the 3 examples, then link to that from the packaged scans and the AF section.

**Comment by sastorsl** [comment]
I've updated the PR accordingly.

**Review by psiinon — APPROVED** [review]

**Comment by psiinon** [comment]
Looks good to me :)

File: site/content/docs/docker/about.md [hunk 1]
```diff
@@ -96,6 +96,25 @@ For more details see the blog posts:
 
 These GitHub actions are a simple way to run the packaged scans, especially if you already use GitHub.
 
+### Mounting the current directory
```
Comment (thc202) [inline site/content/docs/docker/about.md:99]: - `### Mounting the Current Directory`
- IMO better have this as first entry in this section rather in the middle (and use the anchor in the AF entry).

File: site/content/docs/docker/diagnosing-problems.md [hunk 2]
```diff
@@ -41,6 +37,9 @@ If they are then check that any file can be created - the following command shou
 docker run -v $(pwd):/zap/wrk/:rw -t ghcr.io/zaproxy/zaproxy:stable touch /zap/wrk/test.txt
 ```
 
+Note that `$(pwd)` is only supported on Linux and MacOS.
+See [Docker About](/docs/docker/about/#mounting-the-current-directory) for examples for e.g. Windows and Powershell.
+
```
Comment (thc202) [inline site/content/docs/docker/diagnosing-problems.md:42]: This should be moved (and reworded) to where the instructions were before.

File: site/content/docs/docker/about.md (hunk 1 above)
Comment (sastorsl) [reply site/content/docs/docker/about.md:99]: I thought I _did_ use the anchor - am I missing something?

Comment (thc202) [reply site/content/docs/docker/about.md:99]: "See examples above." is where the anchor would be.

Comment (sastorsl) [reply site/content/docs/docker/about.md]: Right. Done.

**Comment by sastorsl** [comment]
One more update added.

**Comment by kingthorin** [comment]
Seems you used a different account, do you want to squash/fixup the commits via interactive  rebase?

**Comment by sastorsl** [comment]
_Now_ I think we're there :-)

File: site/content/docs/docker/about.md (hunk 1 above)
Comment (thc202) [reply site/content/docs/docker/about.md]: The first bullet was not addressed yet.

Comment (sastorsl) [reply site/content/docs/docker/about.md]: A bit cryptic comment, but if you mean Capitalizing the header, sure, by all means. It is not consistent in that document though...

**Comment by sastorsl** [comment]
Are we good now?

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by thc202** [comment]
Closed/reopened to restart the CI check which was stuck.

Leaving the merge if anyone else wants to check the latest changes.

**Review by kingthorin — APPROVED** [review]
Looks good to me

**Pull request merged**